elodie/filesystem.py

```diff
@@ -10,11 +10,33 @@
 import re
 import shutil
 import time
+from os import path
+from configparser import RawConfigParser
 
 from elodie import geolocation
+from elodie import constants
 from elodie import log
 from elodie.localstorage import Db
 
+__DIR__ = None
+
+def get_dir():
```

Comment:
I suggest using a more descriptive name for this function. Something like `get_formatted_directory_name`. I know it's verbose but I think it makes it more clear in the code which calls it.